frontend/src/components/TabNavigation.tsx

```diff
@@ -86,22 +74,26 @@ const TabNavigation = <T extends unknown = undefined>({
     );
   }
 
+  const activeValue = activeItems[0]?.url || navigationTabs[0]?.url || "";
+
   return (
-    <StyledNav variant="tabs">
-      {navigationTabs.map((tab) => (
-        <Nav.Item key={tab.url}>
-          <Link
-            className={tab.isActive ? "nav-link active" : "nav-link"}
-            role="button"
-            tabIndex={0}
-            href={tab.url}
-            data-testid={tab.testId}
-          >
-            {tab.title(intl, tabTitleArguments as T)}
-          </Link>
-        </Nav.Item>
-      ))}
-    </StyledNav>
+    <Tabs.Root
+      value={activeValue}
+      my="4"
```

Comment:
I don't know what `my` stands for. May I suggest using the full names?
Also, where are the Chakra design tokens defined for `4` ?